sdk/eventhub/Azure.Messaging.EventHubs/MigrationGuideFromDotNetFramework.md

```diff
@@ -0,0 +1,696 @@
+
+To try and improve the development experience across Azure services, including Event Hubs, a set of uniform [design guidelines](https://azure.github.io/azure-sdk/general_introduction.html) was created for all languages to drive a consistent experience with established API patterns for all services. A set of [.NET-specific guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html) was also introduced to ensure that .NET clients have a natural and idiomatic feel that mirrors that of the .NET base class libraries. Further details are available in the guidelines for those interested.
+
+The new Event Hubs client library is designed to provide an approachable onboarding experience for those new to messaging and/or the Event Hubs service with the goal of enabling a quick initial feedback loop for publishing and consuming events. A gradual step-up path follows, building on the onboarding experience and shifting from exploration to tackling real-world production scenarios. For developers with high-throughput scenarios or specialized needs, a set of lower-level primitives are available to offer less abstraction and greater control.
+
+While we believe that there is significant benefit to adopting the new Event Hubs client library, it is important to be aware that the legacy version has not been officially deprecated. It will continue to be supported with security and bug fixes as well as receiving some minor refinements. However, in the near future it will not be under active development and new features are unlikely to be added. There is no guarantee of feature parity between the and legacy client library versions.
+
+## Cross-service SDK improvements
+
+The `Azure.Messaging.EventHubs` family of packages also provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as:
+
+-   Using the new `Azure.Identity` library to share a single authentication between clients
+-   A unified diagnostics pipeline offering a common view of the activities across each of the client libraries
+
+## General changes
+
+### Package and namespaces
+
+Package names and the namespace root for the Azure client libraries for .NET have changed. Each will follow the pattern `Azure.[Area].[Service]` where the legacy clients followed the pattern `WindowsAzure.[Service]` or `Microsoft.Azure.[Service]`. This provides a quick and accessible means to help understand, at a glance, whether you are using the current generation or legacy clients.
```

Comment:
Should we mention that the older namespace used was `Microsoft.Messaging.something`?